roles/openshift_health_checker/openshift_checks/etcd_volume_check.py

```diff
@@ -0,0 +1,68 @@
+# pylint: disable=missing-docstring
+"""
+Ansible module for determining information about etcd volume size.
+"""
+
+from openshift_checks import OpenShiftCheck, get_var
+
+
+class EtcdImageData(OpenShiftCheck):
+    """Check that etcd volume capacity is not over 50%"""
+
+    name = "etcd_volume_check"
+    tags = ["etcd_volume_check"]
+
+    def run(self, _, task_vars):
+        ansible_mounts = get_var(task_vars, "ansible_mounts")
+        etcd_mountpath = self.get_etcd_mountpath(ansible_mounts)
+        etcd_diskspace_avail = self.get_etcd_diskspace(etcd_mountpath, ansible_mounts)
+        etcd_totaldiskspace = self.get_etcd_totaldiskspace(etcd_mountpath, ansible_mounts)
```

Comment:
@rhcarvalho 
> Please correct me if I'm mistaken, these seen to be checking the space in the machine where ansible-playbook is run?
>
> I'd expect we be executing an Ansible module only in the hosts that are part of the etcd group.

Double checked by comparing the output of `ansible_mounts` with the output of `lsblk` on the remote host. They certainly seem to match up:

Output of `ansible_mounts`:
```
[{device': /dev/xvda2',
fstype': xfs',
mount': /',
options': rw,seclabel,relatime,attr2,inode64,noquota',
size_available': 21110980608,
size_total': 26830942208},
{'device': /dev/mapper/docker-openshift--xfs--vol--dir',
fstype': xfs',
mount': /mnt/openshift-xfs-vol-dir',
options': rw,seclabel,relatime,attr2,inode64,grpquota',
size_available': 8004644864,
size_total': 8038383616}]
```

Output of `lsblk` on remote host:
```
NAME                                MAJ:MIN RM  SIZE RO TYPE MOUNTPOINT
xvda                                202:0    0   25G  0 disk
├─xvda1                             202:1    0    1M  0 part
└─xvda2                             202:2    0   25G  0 part /
xvdb                                202:16   0   25G  0 disk
└─xvdb1                             202:17   0   25G  0 part
  ├─docker-openshift--xfs--vol--dir 253:0    0  7.5G  0 lvm  /mnt/openshift-xfs-vol-dir
  ├─docker-docker--pool_tmeta       253:1    0   28M  0 lvm
  │ └─docker-docker--pool           253:3    0    7G  0 lvm
  └─docker-docker--pool_tdata       253:2    0    7G  0 lvm
    └─docker-docker--pool           253:3    0    7G  0 lvm
```